src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -67,6 +68,14 @@ public boolean hasPerson(Person person) {
         return persons.contains(person);
     }
 
+    /**
+     * Returns person if a {@code person} with the same name exists in the address book.
+     */
+    public Person matchName(Name name) {
```

Comment:
would findPersonByName be clearer in terms of showing what the return type would be through the method name?